fw/rbcx-coprocessor/src/CdcUartTunnel.cpp

```diff
@@ -99,3 +102,69 @@ void tunnelPoll() {
     tunnelUpstreamHandler();
     tunnelDownstreamHandler();
 }
+
+bool tunnelOnSetLineCoding(
```

Comment:
WIP, tohle ještě bude vyžadovat změny, my totiž musíme při setLineCoding zahodit rx buffer (byl přijat s jiným nastavením UARTu).

Reply:
Uf, nedaří se mi to udělat. On totiž v momentě, kdy přepínáme nastavení tady, už v USB bufferu čeká packet, protože `usbd_ep_write` ten první sežere, a nevidím možnost ho nějak přepsat. Chtělo by to umět detekovat, že ten port je na PC otevřený.

Reply:
Jj něco takovýho asi ANDováno s pinem `VUSB_DET == HIGH` - pro handling odpojení kabelu, což asi knihovna sama neudělá (myslím)

Reply:
Tak nic, to kde jsem linkoval jen nekdo kontroluje jestli je zapnut DTR nebo ne, coz je pitomost. Necham to zatim tak, zajimalo by me, jak to dela ESP32 devkit.